core/src/test/kotlin/admindata/AgentBuildManager.kt

```diff
@@ -0,0 +1,94 @@
+package com.epam.drill.admin.admindata
+
+import com.epam.drill.admin.methods.*
+import com.epam.drill.common.*
+import kotlinx.serialization.protobuf.*
+import kotlin.test.*
+
+class AgentBuildManagerTest {
+    private val buildManager = AgentBuildManager("test")
+
+    @Test
+    fun `init build info with empty version`() {
+        buildManager.initBuildInfo("")
+        assertEquals(buildManager.builds.size, 1)
```

Comment:
expected value should be the first arg of the assertion
```suggestion
        assertEquals(1, buildManager.builds.size)
```